### DISCUSSION TITLE
Is it possible to escape chars in the execute function

### DISCUSSION THREAD

**Discussion by e82eric** [junegunn/fzf#2689] [Q&A] [answered]
Below is the command I am trying to run (in windows).  When I add the + char I get "unknown action: %b)".  I tried using \ and ^ as escape characters and got the same result.  Is there a way to escape the +?

rg --hidden -i --vimgrep "Test" | fzf --bind="enter:execute(for /f \"tokens=1,2 delims=:\" %a in ({}) do nvim %a +%b)

**Comment by junegunn** [comment, marked as answer]
Have you tried using `enter:execute:...` form instead of `enter:execute(...)`?

**Reply by e82eric** [reply]
thanks! that worked perfect.  Is there any way to abort after?

**Reply by junegunn** [reply]
```sh
fzf --bind 'enter:execute:echo hi' --bind 'enter:+abort'
```

Excerpt from `man fzf`

```
   ACTION COMPOSITION
       Multiple actions can be chained using + separator.

       e.g.
            fzf --multi --bind 'ctrl-a:select-all+accept'
            fzf --multi --bind 'ctrl-a:select-all' --bind 'ctrl-a:+accept'
```

**Reply by e82eric** [reply]
thanks! that also worked perfect.  This is probably a windows thing but, but any idea what would cause it to show Press any key to continue when exiting nvim?
